frontend/packages/dev-console/integration-tests/README.md

```diff
@@ -1,4 +1,44 @@
-# This file consists of guide lines to create automation scripts
+# Getting Started
+
+## Directory Structure
+
+Folder structure of cypress cucumber framework
+
+```
+frontend/packages/dev-console/integration-tests/
+├── support              <--- cypress cucumber support configurations
+|   ├── commands         <--- add commands to Cypress 'cy.' global, other support configurations
+|   |   └── index.ts
+|   ├── constants        <--- enums required for dev-console scripts
+│   |   └── add.ts
+|   |   └── global.ts
+|   ├── pages            <--- helper objects and page functions
+│   |   ├── add-flow     <--- Add flow related helper objects and page functions
+|   |   |   └──add-page.ts
+|   |   └── app.ts       <--- Re-usable helper objects and page functions
+|   ├── step-definitions <--- cucumber step implementations
+│   |   ├── common       <--- Re-usable dev-console step definitions
+|   |   |   └──common.ts
+|   |   |   └──project-creation.ts
+|   |   └── app.ts       <--- Re-usable helper objects and page functions
```

Comment:
Don't see `app.ts` under `step-definitions`?

Reply:
In upcoming pr's, this file will be displayed, For now, I will remove this file